src/controllers/wizardfinishpage.js

```diff
@@ -6,20 +6,13 @@ define(["loading"], function (loading) {
         ApiClient.ajax({
             url: ApiClient.getUrl("Startup/Complete"),
             type: "POST"
-        }).then(function () {
-            Dashboard.navigate("dashboard.html");
+        }).then(function() {
```

Comment:
wasn't you adding a space between `function` and `()`?

Reply:
https://github.com/jellyfin/jellyfin-web/pull/509/commits/47cc029c7056b2ad334a93ba80f1568c6020a0ad